src/main/kotlin/no/nav/familie/klage/brev/BrevInnholdUtleder.kt

```diff
@@ -75,58 +76,74 @@ class BrevInnholdUtleder(
-                AvsnittDto(
-                    deloverskrift = "Spørsmålet i saken",
-                    deloverskriftHeading = Heading.H3,
-                    innhold = spørsmåletISaken,
-                ),
-                AvsnittDto(
-                    deloverskrift = "Aktuelle rettskilder",
-                    deloverskriftHeading = Heading.H3,
-                    innhold = aktuelleRettskilder,
-                ),
-                AvsnittDto(
-                    deloverskrift = "Klagers anførsler",
-                    deloverskriftHeading = Heading.H3,
-                    innhold = klagersAnførsler,
-                ),
-                AvsnittDto(
-                    deloverskrift = "Vurdering av klagen",
-                    deloverskriftHeading = Heading.H3,
-                    innhold = vurderingAvKlagen,
+                listOfNotNull(
+                    AvsnittDto(
+                        deloverskrift = "",
+                        innhold =
+                            "Vi har ${klageMottatt.norskFormatLang()} fått klagen din på vedtaket om " +
+                                "${visningsnavn(stønadstype, påklagetVedtakDetaljer)} som ble gjort " +
+                                "${påklagetVedtakDetaljer.vedtakstidspunkt.norskFormatLang()}, " +
+                                "og kommet frem til at vi ikke endrer vedtaket. Nav Klageinstans skal derfor vurdere saken din på nytt.",
+                    ),
+                    AvsnittDto(
+                        deloverskrift = "",
+                        innhold = "Saksbehandlingstidene finner du på nav.no/saksbehandlingstider.",
+                    ),
+                    AvsnittDto(
+                        deloverskrift = "Dette er vurderingen vi har sendt til Nav Klageinstans",
+                        deloverskriftHeading = Heading.H2,
+                        innhold = "",
+                    ),
+                    *(
+                        klagefristUnntakBegrunnelse?.let {
+                            listOf(
+                                AvsnittDto(
+                                    deloverskrift = "Dokumentasjon og utredning",
+                                    deloverskriftHeading = Heading.H3,
+                                    innhold = klagefristUnntakBegrunnelse,
+                                ),
+                                AvsnittDto(
+                                    deloverskrift = "",
+                                    innhold = dokumentasjonOgUtredning,
+                                ),
+                            ).toTypedArray()
+                        } ?: listOf(
+                            AvsnittDto(
+                                deloverskrift = "Dokumentasjon og utredning",
+                                deloverskriftHeading = Heading.H3,
+                                innhold = dokumentasjonOgUtredning,
+                            ),
+                        ).toTypedArray()
+                    ),
```

Comment:
Jeg vet ikke hva som er lettest å lese, men du kunne vurdert å skrive det på denne måten, for å slippe å gjøre om til `TypedArray` og bruke spread operator, og slippe duplikat av `deloverskrift` og `deloverskriftHeading` 🤷🏼 

```kotlin
AvsnittDto(
    deloverskrift = "Dokumentasjon og utredning",
    deloverskriftHeading = Heading.H3,
    innhold = klagefristUnntakBegrunnelse ?: dokumentasjonOgUtredning,
),
klagefristUnntakBegrunnelse?.let {
    AvsnittDto(
        deloverskrift = "",
        innhold = dokumentasjonOgUtredning,
    )
},
```

Dersom du velger å beholde det sånn det er nå trenger vel ikke listen på linje 79 å være `NotNull`?